src/styles/card.scss

```diff
@@ -0,0 +1,112 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: $card-width;
+  border-radius: $border-radius;
+  border: 1px solid $border-color;
+
+  &__product-image {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 40px;
+  }
+
+  &__product-details {
+    width: $card-content-width-adjustment;
+    margin-inline: $padding-inline;
+    margin-bottom: $margin-bottom;
+
+    &-title {
+      height: 36px;
+      font-size: $font-size-medium;
+      font-weight: $font-weight-medium;
+      line-height: 1.5;
+      text-align: left;
+      color: $secondary-color;
+      margin-top: 0;
+      margin-bottom: 4px;
+    }
+
+    &-description {
+      margin: 0;
+      font-size: $font-size-small;
+      line-height: 14px;
+      text-align: left;
+      color: $text-color;
+    }
+  }
+
+  &__rating {
+    box-sizing: border-box;
+    width: $card-content-width-adjustment;
+    height: 16px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    margin-inline: $padding-inline;
+    margin-bottom: 24px;
+
+    &-reviews {
+      font-size: $font-size-small;
+      line-height: 14px;
+      text-align: right;
+      color: $secondary-color;
+    }
+  }
+
+  &__price {
+    box-sizing: border-box;
+    width: $card-content-width-adjustment;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    margin-inline: $padding-inline;
+    margin-bottom: $margin-bottom;
+
+    &-title {
+      margin: 0;
+      font-size: $font-size-medium;
+      line-height: 18px;
+      text-align: left;
+      color: $text-color;
+    }
+
+    &-cost {
+      margin: 0;
+      font-size: $font-size-large;
+      font-weight: $font-weight-bold;
+      line-height: 18px;
+      text-align: right;
+      color: $secondary-color;
+    }
+  }
+
+  &__button-buy {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 166px;
+    height: 40px;
```

Comment:
The width and height of the 'button-buy' class are hard-coded. It's recommended to use variables for these values to maintain consistency and to facilitate easier adjustments in the future.